src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -363,30 +363,11 @@ Thumbnails =
 # Default: false
 ThumbnailPadding = 
 
-# Some renderers (e.g. Sony Bravia TVs and Blu-ray Disc players) require
-# thumbnails to be represented as JPEGs in the DLNA directory XML. Set this to
-# true to force UMS to fake this representation.
-# Note: this has no effect on real thumbnail file formats or content-type
-# headers, and will only return true if MediaInfo = true
-# Default: false
-ForceJPGThumbnails = 
-
-# Whether the renderer (e.g. Samsung TVs) requires the use of the "res" element
-# instead of the "albumArtURI" element for thumbnails in DLNA reponses.
-# element.
-# Default: false, i.e. use "albumArtURI"
-ThumbnailAsResource = 
```

Comment:
All thumbnails are offered both as "resource" ```<res>``` and as ```albumArtURI``` even though it actually breaks the standard. According to DLNA, ```albumArtURI``` should only be used on ```audioItem``` and ```musicItem```. "resource" is the default thumbnail type, and should be offered for all types. However, to avoid breaking anything for renderers that potentially doesn't adhere to the standard, I figured that the safest way was to offer both for all media types.